### PR TITLE
RHOARDOC-1667 ensure _stand-alone_ is always hyphenated

### PR DESCRIPTION
**Pull request by inoxx03**

### CODE REVIEW

File: docs/topics/assembly_deploying-runtime-application-to-stand-alone-red-hat-enterprise-linux.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// As of this writing, this assembly is common for
+// Springboot, Thorntail, Vertx 
+[id='deploying-runtime-application-to-stand-alone-red-hat-enterprise-linux_{context}']
+= Deploying {runtime} application tostand-aloneRed Hat Enterprise Linux
+
+:parent-context: {context}
+To deploy your {runtime} application tostand-aloneRed{nbsp}Hat Enterprise Linux, configure the `pom.xml` file in the application, package it using Maven and deploy using the `java -jar` command.
```
Comment (huntley) [inline docs/topics/assembly_deploying-runtime-application-to-stand-alone-red-hat-enterprise-linux.adoc]: Missing space before and after stand-alone

File: docs/topics/assembly_deploying-runtime-application-to-stand-alone-red-hat-enterprise-linux.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+= Deploying {runtime} application tostand-aloneRed Hat Enterprise Linux
+
+:parent-context: {context}
+To deploy your {runtime} application tostand-aloneRed{nbsp}Hat Enterprise Linux, configure the `pom.xml` file in the application, package it using Maven and deploy using the `java -jar` command. 
+
+.Prerequisites
+
+* RHEL 7 or RHEL 8 installed.
+
+// Preparing runtime application forstand-alonered hat enterprise linux deployment
```
Comment (huntley) [inline docs/topics/assembly_deploying-runtime-application-to-stand-alone-red-hat-enterprise-linux.adoc]: In a comment but missing space before and after stand-alone

File: docs/topics/assembly_deploying-runtime-application-to-stand-alone-red-hat-enterprise-linux.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+include::proc_preparing-runtime-application-for-stand-alone-red-hat-enterprise-linux-deployment.adoc[leveloffset=+1]
+
+// Deploying runtime application tostand-alonered hat enterprise linux
```
Comment (huntley) [inline docs/topics/assembly_deploying-runtime-application-to-stand-alone-red-hat-enterprise-linux.adoc]: In a comment but missing space before and after stand-alone

File: docs/topics/assembly_developing-and-deploying-a-nodejs-application.adoc [hunk 4]
```diff
@@ -11,7 +11,7 @@ include::proc_developing-a-nodejs-application.adoc[leveloffset=+1]
 // Deploying a Nodejs application to OpenShift
 include::assembly_deploying-a-nodejs-application-to-openshift.adoc[leveloffset=+1]
 
-// Deploying a Nodejs application to Standalone Red Hat Enterprise Linux
-include::proc_deploying-a-node-js-application-to-standalone-red-hat-enterprise-linux.adoc[leveloffset=+1]
+// Deploying a Nodejs application tostand-aloneRed Hat Enterprise Linux
```
Comment (huntley) [inline docs/topics/assembly_developing-and-deploying-a-nodejs-application.adoc]: Missing space before and after stand-alone

**Comment by inoxx03** [comment]
I've added hyphenation to all instances of the word _stand-alone_ in our docs to ensure consistency with the IBM Style Guide. I have not edit any of the Thorntail imported modules i thids PR, I'll do that in a seprate PR on the `thorntail-prod` repo. @huntley Please review and let me know if you have comments or suggestions. Thank you.

File: docs/topics/assembly_developing-and-deploying-springboot-application.adoc [hunk 5]
```diff
@@ -31,8 +31,8 @@ include::proc_developing-runtime-application.adoc[leveloffset=+1]
 // Deploying Springboot application to OpenShift
 include::assembly_deploying-runtime-application-to-openshift.adoc[leveloffset=+1]
 
-// Deploying Springboot application to standalone Red Hat Enterprise Linux 
-include::assembly_deploying-runtime-application-to-standalone-red-hat-enterprise-linux.adoc[leveloffset=+1]
+// Deploying Springboot application tostand-aloneRed Hat Enterprise Linux
```
Comment (huntley) [inline docs/topics/assembly_developing-and-deploying-springboot-application.adoc]: In a comment, but missing space before and after stand-alone

File: docs/topics/assembly_developing-and-deploying-thorntail-application.adoc [hunk 6]
```diff
@@ -21,8 +21,8 @@ include::thorntail/docs/howto/writing-an-application-from-scratch/index.adoc[lev
 // Deploying runtime application to openshift
 include::assembly_deploying-runtime-application-to-openshift.adoc[leveloffset=+1]
 
-// Deploying runtime application to standalone red hat enterprise linux
-include::assembly_deploying-runtime-application-to-standalone-red-hat-enterprise-linux.adoc[leveloffset=+1]
+// Deploying runtime application tostand-alonered hat enterprise linux
```
Comment (huntley) [inline docs/topics/assembly_developing-and-deploying-thorntail-application.adoc]: In a comment, but missing space before and after stand-alone

File: docs/topics/assembly_developing-and-deploying-vertx-application.adoc [hunk 7]
```diff
@@ -3,7 +3,7 @@
 = Developing and deploying {VertX} runtime application
 :parent-context: {context}
 
-In addition to xref:example-rest-http-vertx[using an example], you can create a new {VertX} application from scratch and deploy it to OpenShift or standalone Red{nbsp}Hat Enterprise Linux.
+In addition to xref:example-rest-http-vertx[using an example], you can create a new {VertX} application from scratch and deploy it to OpenShift orstand-aloneRed{nbsp}Hat Enterprise Linux.
```
Comment (huntley) [inline docs/topics/assembly_developing-and-deploying-vertx-application.adoc]: Missing space before and after stand-alone

File: docs/topics/con_available-runtimes.adoc [hunk 8]
```diff
@@ -12,7 +12,7 @@ endif::[]
 +
 See link:{link-guide-vertx}[{name-guide-vertx}] for information about {Vertx} application development.
 
-{Thorntail}:: A framework based on the popular {WildFly} Java application server to enable the creation of small, standalone microservice-based applications.
+{Thorntail}:: A framework based on the popular {WildFly} Java application server to enable the creation of small,stand-alonemicroservice-based applications.
```
Comment (huntley) [inline docs/topics/con_available-runtimes.adoc]: Missing space before and after stand-alone

File: docs/topics/proc_deploying-a-node-js-application-to-stand-alone-red-hat-enterprise-linux.adoc [hunk 9]
```diff
@@ -1,7 +1,7 @@
-[id='deploying-a-node-js-application-to-standalone-red-hat-enterprise-linux_{context}']
-= Deploying a {Node} application to standalone Red Hat Enterprise Linux
+[id='deploying-a-node-js-application-to-stand-alone-red-hat-enterprise-linux_{context}']
+= Deploying a {Node} application tostand-aloneRed Hat Enterprise Linux
```
Comment (huntley) [inline docs/topics/proc_deploying-a-node-js-application-to-stand-alone-red-hat-enterprise-linux.adoc]: Missing space before and after stand-alone

File: docs/topics/proc_deploying-a-node-js-application-to-stand-alone-red-hat-enterprise-linux.adoc [hunk 10]
```diff
@@ -1,7 +1,7 @@
...
 
-You can deploy a {Node} application to standalone Red Hat Enterprise Linux using `npm`.
+You can deploy a {Node} application tostand-aloneRed Hat Enterprise Linux using `npm`.
```
Comment (huntley) [inline docs/topics/proc_deploying-a-node-js-application-to-stand-alone-red-hat-enterprise-linux.adoc]: Missing space before and after stand-alone

File: docs/topics/proc_deploying-runtime-application-to-stand-alone-red-hat-enterprise-linux-using-jar.adoc [hunk 11]
```diff
@@ -9,10 +9,10 @@
 //  context: used in anchor IDs to conflicts due to duplicate IDs.
 //
 // Rationale: This procedure is the same for 2 or more runtimes.
-[id='deploying-runtime-application-to-standalone-red-hat-enterprise-linux-using-jar_{context}']
-= Deploying {runtime} application to standalone Red Hat Enterprise Linux using jar
+[id='deploying-runtime-application-to-stand-alone-red-hat-enterprise-linux-using-jar_{context}']
+= Deploying {runtime} application tostand-aloneRed Hat Enterprise Linux using jar
```
Comment (huntley) [inline docs/topics/proc_deploying-runtime-application-to-stand-alone-red-hat-enterprise-linux-using-jar.adoc]: Missing space before and after stand-alone

File: docs/topics/proc_deploying-runtime-application-to-stand-alone-red-hat-enterprise-linux-using-jar.adoc [hunk 12]
```diff
@@ -9,10 +9,10 @@
...
 
-To deploy your {runtime} application to standalone Red{nbsp}Hat Enterprise Linux, use `java -jar` command.
+To deploy your {runtime} application tostand-aloneRed{nbsp}Hat Enterprise Linux, use `java -jar` command.
```
Comment (huntley) [inline docs/topics/proc_deploying-runtime-application-to-stand-alone-red-hat-enterprise-linux-using-jar.adoc]: Missing space before and after stand-alone

File: docs/topics/proc_preparing-runtime-application-for-stand-alone-red-hat-enterprise-linux-deployment.adoc [hunk 13]
```diff
@@ -9,10 +9,10 @@
 //  context: used in anchor IDs to conflicts due to duplicate IDs.
 //
 // Rationale: This procedure is the same for 2 or more runtimes.
-[id='preparing-runtime-application-for-standalone-red-hat-enterprise-linux_{context}']
-= Preparing {runtime} application for standalone Red Hat Enterprise Linux deployment
+[id='preparing-runtime-application-for-stand-alone-red-hat-enterprise-linux_{context}']
+= Preparing {runtime} application forstand-aloneRed Hat Enterprise Linux deployment
```
Comment (huntley) [inline docs/topics/proc_preparing-runtime-application-for-stand-alone-red-hat-enterprise-linux-deployment.adoc]: Missing space before and after stand-alone

File: docs/topics/proc_preparing-runtime-application-for-stand-alone-red-hat-enterprise-linux-deployment.adoc [hunk 14]
```diff
@@ -9,10 +9,10 @@
...
 
-For deploying your {runtime} application to standalone Red Hat Enterprise Linux, you must first package the application using Maven.
+For deploying your {runtime} application tostand-aloneRed Hat Enterprise Linux, you must first package the application using Maven.
```
Comment (huntley) [inline docs/topics/proc_preparing-runtime-application-for-stand-alone-red-hat-enterprise-linux-deployment.adoc]: Missing space before and after stand-alone

File: docs/topics/ref_glossary.adoc [hunk 15]
```diff
@@ -4,7 +4,7 @@
 
 == Product and project names
 
-{name-launcher-oso}:: link:{link-launcher-oso}[{name-launcher-oso}] is a standalone getting started experience offered by Red Hat for jumpstarting cloud-native application development on OpenShift. It provides a way of creating functional example applications as well as an easy way to build and deploy these example applications on OpenShift. 
+{name-launcher-oso}:: link:{link-launcher-oso}[{name-launcher-oso}] is astand-alonegetting started experience offered by Red Hat for jumpstarting cloud-native application development on OpenShift. It provides a way of creating functional example applications as well as an easy way to build and deploy these example applications on OpenShift.
```
Comment (huntley) [inline docs/topics/ref_glossary.adoc]: Missing space before and after stand-alone

**Comment by inoxx03** [comment]
@huntley Thank you. What I bang-up job I've done searching and replacing :D I'll address your comments ASAP.

**Comment by inoxx03** [comment]
I've corrected the damage done by an earlier search-and-replace mess-up. @huntley Please re-review and let me know if you spot something I missed:)

**Comment by inoxx03** [comment]
@huntley. Thank you for the re-review and ack:)

**Pull request merged**